backend/.env.example

```diff
@@ -3,3 +3,6 @@ DBPASSWORD=password
 DBHOST=localhost
 DBPORT=5018
 DATABASENAME=booking_dev
+SENDER=kvitteringsbot@online.ntnu.no
+EMAILSERVICECLIENT=""
+EMAILPRIVATEKEY=""
```

Comment:
I'm assuming this is information which will be changed for the production build

(same goes for any other secret information in .env files)

Reply:
Yeah, i don't have the right user/email addresses created yet to be able to fill those fields